apps/src/dance/DanceAiModal.tsx

```diff
@@ -144,25 +191,122 @@ const DanceAiModal: React.FunctionComponent<DanceAiProps> = ({onClose}) => {
       response
     );
     const pickedResponseJson = JSON.stringify(pickedResponse);
-    const formattedPickedResponseJson = pickedResponseJson
-      .replace(/":/g, '": ')
-      .replace(/","/g, '", "')
-      .replace(/,"/g, ', "');
 
     // The block value will be set to this JSON.
     setResponseJson(pickedResponseJson);
 
-    // The user will see this function call.
-    setResponseCall(`ai(${formattedPickedResponseJson})`);
-
     // The user will see this explanation.
     setResponseExplanation(response.explanation);
   };
 
+  const convertBlocks = () => {
```

Comment:
This feels like a good function to hoist out into a utils or danceBlocklyUtils or something?

Reply:
Good call.  Will rearrange this and more files in the next PR.